nncf/api/compression.py

```diff
@@ -182,6 +182,22 @@ def scheduler(self) -> CompressionScheduler:
         :return: The instance of the `CompressionScheduler`.
         """
 
+    def load_state(self, state: Dict[str, object]) -> None:
+        """
+        Loads the compression controller state.
+
+        :param state: Output of `get_state()` method.
+        """
+        self.scheduler.load_state(state)
+
+    def get_state(self):
```

Comment:
`get_state` should be an abstract method. Please add a type hint for the return value.

Reply:
Done